cosmo/monitors/data_models.py

```diff
@@ -192,3 +194,57 @@ def get_new_data(self):
         data_results = data_results.replace([np.inf, -np.inf], np.nan).dropna().reset_index(drop=True)
 
         return data_results[~data_results.EXPTYPE.str.contains('ACQ|DARK|FLAT')]
+
+
+class FUVDarkDataModel(BaseDataModel):
+    files_source = FILES_SOURCE
+    subdir_pattern = '?????'
+    cosmo_layout = False
+
+    # program_id = ['15771/', '15533/', '14940/', '14520/', '14436/', '13968/', '13521/', '13121/', '12716/', '12423/',
+    #               '11895/']
+
+    primary_key = 'ROOTNAME'
+
+    def get_new_data(self):
+        """Retrieve data"""
+        header_request = {
+            0: ['ROOTNAME', 'SEGMENT'],
+            1: ['EXPTIME', 'EXPSTART']
+        }
+
+        table_request = {
+            1: ['PHA', 'XCORR', 'YCORR', 'TIME'],
+            3: ['TIME', 'LATITUDE', 'LONGITUDE']
+        }
+        """
+        files = []
+
+        for prog_id in self.program_id:
+
+            new_files_source = os.path.join(FILES_SOURCE, prog_id)
+            files += find_files('*corrtag*', data_dir=new_files_source, subdir_pattern=self.subdir_pattern)
+
+        if self.model is not None:
+            currently_ingested = [item.FILENAME for item in self.model.select(self.model.FILENAME)]
+
+            for file in currently_ingested:
+                files.remove(file)
+        """
+        files = []
+        program_ids = ['15771/']
+        for program in program_ids:
+            new_files_source = os.path.join(FILES_SOURCE, program)
+            subfiles = glob(os.path.join(new_files_source, "*corrtag*"))
```

Comment:
You can use find files if you make the changes mentioned in https://github.com/spacetelescope/cosmo/pull/73/files#r379640719

But this is essentially doing the same thing, so up to you!